Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/rhscl/ruby-23-rhel7
+
+# sshd
+USER root
+RUN yum install -y --setopt=tsflags=nodocs openssh-server libicu-devel && \
+    yum clean all && /usr/sbin/sshd-keygen
+
+RUN mkdir /var/run/sshd
+
+# TODO: is this needed? https://git.io/vrplM
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+
+# gems
+RUN ["bash", "-c", "gem install --no-ri --no-rdoc bunny"]
+
+# git user
+RUN groupadd -r git &&\
+     useradd -r -s /bin/bash -g git git && \
+     mkdir --parent /home/git && \
+     chown -R git:git /home/git
+
+# gitlab-shell setup
+COPY . /home/git/gitlab-shell
+WORKDIR /home/git/gitlab-shell
+RUN ["bash", "-c", "./bin/install"]
+
+COPY authorized_keys /home/git/.ssh/authorized_keys
+
+USER root
```

Comment:
Yeah, this can be removed. It's a leftover from a previous iteration, when there was a `USER git` line earlier. I'm not sure that we can change it back to a non-root user though: `sshd` is intended to be run, which I tried as non-root and it wouldn't start (maybe it could be hacked around?).

Hmm, just thinking out loud: what would happen if here in the Dockerfile we did something like `RUN systemctl start sshd` or whatever, as root; then switched back to an unprivileged user and set something else as the `CMD`? I guess that probably wouldn't work and even if `sshd` did try to start, we'd still need to run the container as root?